.github/workflows/slack-workflow-run.yml

```diff
@@ -1,16 +1,17 @@
 name: Slack Workflow Run
 
 on:
-  workflow_run:
```

Comment:
This approach seems to bring more complexity, no?

Reply:
I would say it adds more lines of code but it reduces complexity. I think the flow is a little clearer now - and will give us more flexibility later on if we want to add alerts to new workflows. But yeah, it's still more complex than I would like it too...